jormungandr/Cargo.toml

```diff
@@ -73,6 +73,11 @@ features = ["rustls-tls"]
 [dev-dependencies]
 rand_core = "0.5"
 tokio = { version = "^1.0", features = ["full"] }
+quickcheck = "0.9"
+quickcheck_macros = "0.9"
```

Comment:
It's better to use quickcheck 1.0 while we are at it, possibly integrating #2927, https://github.com/input-output-hk/chain-libs/pull/506, and https://github.com/input-output-hk/chain-libs/pull/507. Unless this is too much work and requires another PR.

Reply:
I think it does require another PR, there are breaking interface changes.